osidb/mixins.py

```diff
@@ -295,6 +295,10 @@ def get_embargoed_acl():
     def is_embargoed(self):
         return self.acl_read == ACLMixin.get_embargoed_acl()
 
+    @property
+    def is_internal(self):
+        return {self.acl_read[0], self.acl_write[0]} == self.acls_internal
```

Comment:
Why do we care only for the first items of the arrays? This method should be IMO general and compare everything and not care for how many items are there. Maybe it is currently one member in each field but we do not need that information here.

On the other hand if you want to make it simple you IMO do not have to care for the write part. The visibility is only defined by the read part (sure that currently they must correspond).